.vscode/settings.json

```diff
@@ -21,5 +21,12 @@
 	},
 	// Enable Lint and format using Biome
 	"biome.enabled": true,
-	"editor.defaultFormatter": "biomejs.biome"
+	"editor.defaultFormatter": "biomejs.biome",
+	"editor.codeActionsOnSave": {
+		"source.fixAll": "always",
```

Comment:
I am not opposed to this but sometimes it leads to weird issues when biome forces you know something is clearly wrong and then you keep getting surprised by the over writing on every save.

Reply:
@arafatkatze  are you referring to the `source.fixAll.biome` option specifically? Do you have an example on when this happen?

Reply:
Of the top of my head no, but I did see that happen before sometime. 

LEt's all this for now and see how people experience this.